### PR TITLE
Styling hotfix

### LINKED ISSUE #59: styling fixes
- [x] cocktail show page, button under image to be separated from image and centered
- [x] margarita is not stupid big
- [ ] background for logged in user
- [ ] dashboard huge button

### PR DESCRIPTION
**Pull request by twarbelow**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [x] Other (please describe): styling
## What is the current behavior?

Issue Number: 
closes #59 
## What is the new behavior?

- nothing exciting
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by markcyen — APPROVED** [review]

File: src/components/cocktail.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default class Cocktail extends React.Component {
         <div className="flex justify-around">
   
           <div className="font-montserrat font-semibold">
-            <div className="cocktail_image w-96 h-96 bg-cover" style={{backgroundImage: `url("${thumbnail}")`}}></div>
+            <div className="cocktail_image w-96 h-96 mb-4 bg-cover" style={{backgroundImage: `url("${thumbnail}")`}}></div>
```
Comment (mkrumholz) [inline src/components/cocktail.js:64]: Thank you!! Spacing here looks glorious now ✨

**Review by redferret — APPROVED** [review]

File: src/components/auth_panel.js [hunk 2]
```diff
@@ -6,9 +6,10 @@ export default class AuthPanel extends React.Component {
     return (
       <div className='grid grid-cols-1 gap-2 place-items-center font-playfair'>
         <div className='text-7xl p-10 text-center text-white'>What should I drink tonight?</div>
-        <div className='px-10 pb-5 text-white'>Maybe it’s an old classic or maybe it’s something completely brand new. Tell us what you’re into and we’ll suggest something for you.</div>
+        <div className='px-10 text-white'>Maybe it’s a classic or maybe it’s something new.</div>
+        <div className='px-10 pb-5 text-white'>Tell us what you’re into and we’ll suggest something for you.</div>
```
Comment (mkrumholz) [inline src/components/auth_panel.js:10]: Jussst made a tiny change to actually give this the limericky feel it wanted

File: src/components/logo.js [hunk 3]
```diff
@@ -3,11 +3,14 @@ import React from 'react';
...
-        <p className="w-max opacity-50">drink this</p>
-      </div>
+      <a href='/dashboard'>
+        <div className="border-4 border-black w-min m-8 p-2 font-playfair font-black text-sm">
+          <p className="w-max">drink this</p>
+          <p className="w-max opacity-70">drink this</p>
+          <p className="w-max opacity-50">drink this</p>
+        </div>
+      </a>
+
```
Comment (mkrumholz) [inline src/components/logo.js:13]: YAY glad this is linked now!

File: src/components/starter_cocktails.js [hunk 4]
```diff
@@ -35,12 +34,9 @@ export default class StarterCocktails extends React.Component {
...
         <div className="flex justify-center space-x-8">
           {data.map((cocktail,index) => {
-            // if (cocktail.attributes.rating > 0 && !this.state.clicked.includes(index)) {
-            //   this.setState({clicked: [...this.state.clicked, index]})
-            // } 
             return (
-              <div className="text-center" id={index} key={cocktail.attributes.name}>
-                <img src={cocktail.attributes.thumbnail} alt="Cocktail" />
+              <div className="text-center" id={index} key={cocktail.name}>
+                <img className='h-60 w-60' src={cocktail.attributes.thumbnail} alt="Cocktail" />
```
Comment (mkrumholz) [inline src/components/starter_cocktails.js:39]: Nice way to deal with the massive marg challenge

**Review by mkrumholz — APPROVED** [review]
Nice work! Pulled down and messed around. Added a link and a quick text change, but otherwise this looks good to me.

**Pull request merged**